packages/web3-eth-accounts/src/account.ts

```diff
@@ -77,3 +224,78 @@ export const create = (): {
 		encrypt,
 	};
 };
+
+/**
+ *  Decrypts a v3 keystore JSON, and creates the account.
+ *
+ * */
+export const decrypt = async (
+	keystore: Keystore | string,
+	password: string,
```

Comment:
We should be uniform in our support of string and buffers, so we should probably add the ability to pass a buffer here